### ISSUE TITLE
recent large change to RDF rendering

### ISSUE THREAD

**Issue by balhoff** [biolink/biolink-model#421]
In a recent commit (I think 88d200d492f1f4b2efc8a1a0ab6dbaa42f157328) there was a drastic change to the RDF rendering of Biolink model: all the terms became blank nodes and resource IRIs became string values. Here is a small example:

Previous:

```turtle
<https://w3id.org/biolink/vocab/SensitivityQuantifier> a meta:ClassDefinition ;
    skos:inScheme <https://w3id.org/biolink/biolink-model> ;
    meta:class_uri <https://w3id.org/biolink/vocab/SensitivityQuantifier> ;
    meta:is_a <https://w3id.org/biolink/vocab/RelationshipQuantifier> ;
    meta:mixin true .
```

Current:

```turtle
[ a metatype:ClassDefinition ;
            rdfs:label "SensitivityQuantifier" ;
            metatype:class_uri "https://w3id.org/biolink/vocab/SensitivityQuantifier" ;
            metatype:definition_uri "https://w3id.org/biolink/vocab/SensitivityQuantifier" ;
            metatype:from_schema "https://w3id.org/biolink/biolink-model" ;
            metatype:is_a "RelationshipQuantifier" ;
            metatype:mixin true ]
```

Is this intended? The new format is much less useful.

**Comment by deepakunni3** [comment]
I don't think this is by design.

CC'ing @hsolbrig

**Comment by hsolbrig** [comment]
Nope - on it.

**Comment by hsolbrig** [comment]
Ok - long answer:
1) I made a mistake in the script -- the gen-rdf line should use `biolink/context.jsonld`, because we are emitting instances of biolinkml.  We need to change that back.  That (once the issue below is fixed) 
2) We have an error in biolinkml:  `subproperty_of` has a range of uriorcurie.  The following:
```yaml
genotype to genotype part association:
    is_a: association
    defining_slots:
      - subject
      - object
    description: >-
      Any association between one genotype and a genotypic entity that is a sub-component of it
    slot_usage:
      relation:
        subproperty_of: has variant part
```
Should never have passed, as "has variant part" is NOT a URI or CURIE.   We will fix this in the biolinkml code.

This leads to a second question, however -- should the range of subproperty_of be SlotDefinition or `uriorcurie`?  What IS the intent of 'has variant part' above?

**Comment by hsolbrig** [comment]
To follow up: either (a) `subproperty_of` needs to have a range of `ClassDefinition`? `SlotDefinition`? or we need to practice good identifier hygiene and change the above to `subproperty_of: has_variant_part`

**Comment by balhoff** [comment]
@deepakunni3 could a release be made using a prior biolinkml release? We can't use the current biolink model release.

**Comment by hsolbrig** [comment]
Why not?

**Comment by balhoff** [comment]
We rely on biolink-model.ttl, and it changed as above.

**Comment by hsolbrig** [comment]
Ah - gotcha.  Maybe we can coordinate the changes? 

A couple of options:
1) Comment out subproperty_of for this release
2) Declare slots for the targets of subproperty of, and We can have a revised biolinkml ready to go in a couple o hours...

**Comment by hsolbrig** [comment]
@balhoff -- see slack ccdh channel chat

**Comment by hsolbrig** [comment]
See: https://github.com/biolink/biolinkml/blob/master/tests/test_biolink_model/input/biolink-model.yaml for proposed changes to biolink-model.yaml.    https://github.com/biolink/biolinkml/blob/master/tests/test_biolink_model/output/biolink-model.ttl contains the generated RDF.

If this is OK, we will issue v1.5.9 in PyPi

**Comment by nlharris** [comment]
Can this be closed?

**Comment by balhoff** [comment]
Still waiting for #434 to be merged.

**Comment by balhoff** [comment]
Fixed by #491.
